docs-chef-io/content/inspec/attestations.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Attestations"
+draft = false
+gh_repo = "inspec"
+
+[menu]
+  [menu.inspec]
+    title = "Attestations"
+    identifier = "inspec/reference/attestations.md Attestations"
+    parent = "inspec/reference"
+    weight = 140
++++
+
+Attestation is a mechanism to mark `Not Reviewed (N/R)` controls to `passed` or `failed` manually using an attestation file.An attestation file identifies:
+
+1. which controls needs to be attested.
+2. an explanation of why it is manually attested.
+3. control status `passed` or `failed` to attest controls.
+3. (optionally) an URL pointing to website containing information on control attestation.
+4. (optionally) an expiration date of attestation.
+
+## Usage
+
+To use waivers, you must have a correctly formatted attestation file and
+invoke `inspec exec` with `--attestation-file [path]`.
+
+```bash
+inspec exec path/to/profile --attestation-file attestation.yaml
+```
+
+## File Format
+
+Attestation files support YAML, JSON, CSV, XLSX & XLS format.
```

Comment:
Need docs for CSV and XLSX formats

Reply:
How can we include CSV or XLSX example here?

Reply:
List the headers that are expected. For XLS, a screenshot would be helpful.